css/main.css

```diff
@@ -307,21 +307,153 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+/* button styles */
+.btn {
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #f55800;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  padding: 8px 10px;
+  width: fit-content;
+  transition: all 0.3s ease-in-out 0s;
+  display: flex;
+  align-items: center;
+}
+
+.btn:hover {
+  background: #f55800;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  cursor: pointer;
+}
+
+.btn:disabled {
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+/* Contact Form */
+.footer {
+  background-color: #fff;
+  border-top-left-radius: 90px;
+}
+
+.footer .contact {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 100px 0;
+}
+
+.footer .contact h2 {
+  margin-top: 0;
+}
+
+.footer .contact h2,
+.footer .contact p {
+  text-align: initial;
+  color: #ebebff;
+}
+
+.footer .contact p {
+  font-size: 40px;
+  line-height: 52px;
+  transition: transform 2s ease-in-out 0s;
+  color: #262626;
+  font-family: Inter, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+.footer .contact p:hover {
+  transform: translate(20px);
+}
+
```

Comment:
- [ ] Required: Kindly pls make sure you make the footer paragraph in the centre as per Figma's design, colours, fonts and pattern for the mobile version.
This is your contact form design- 
<img width="501" alt="Screenshot 2022-03-09 at 11 42 41 PM" src="https://user-images.githubusercontent.com/70269507/157536474-fc1abdf3-338f-4896-8246-ccee2cc175d8.png">
This is the Figma one which you should follow-
<img width="267" alt="Screenshot 2022-03-10 at 12 12 00 AM" src="https://user-images.githubusercontent.com/70269507/157536867-0d23646b-8b5a-490e-a8ea-908e056c63cc.png">